src/store/reducers/drinks.js

```diff
@@ -0,0 +1,59 @@
+import * as actions from '../actions/actionTypes';
+
+const initialState = {
+  loading: false,
+  drinks: []
+}
+
+const setDrinks = (state, action) => {
+  
+  const results = action.drinks.map(drink => {
+    let id = drink.idDrink;
```

Comment:
destructuring in this 3 variables this will safe the other 2 lines of code

Reply:
also i will hit you in the head hahaha, please, you are using to much let when the let cases are pretty low compared to const, this is related to performance. ONLY USE LET IF IT WILL CHANGE, IF NOT USE CONST. take that in mind for every thing you do and all the files in this pr. if you have some confussion between when and when not to use it, consider reviewing this topic about block variables.